app/services/call_creator.rb

```diff
@@ -91,4 +112,15 @@ def add_error(key, message)
     @errors[key] ||= []
     @errors[key] << message
   end
+
+  def manual_target
+    {
+      country_code: @params[:target_country_code],
+      phone_number: @params[:target_phone_number],
+      phone_extension: @params[:target_phone_extension],
+      name: @params[:target_name],
+      title: @params[:target_title] || '',
+      caller_id: nil # autofilled with caller's number
+    }
+  end
```

Comment:
So perhaps we'd have a function that abstracts away how we decide the targets. We either supply the page's targets list, or the manual target.